scoring.js

```diff
@@ -0,0 +1,58 @@
+const calculateScore = (player) => {
+  switch (player.position) {
+    case 'QB':
+      return passingScore(player) + rushingScore(player)
+    case 'RB':
+      return rushingScore(player) + receivingScore(player) + returnScore(player)
+    case 'WR':
+      return rushingScore(player) + receivingScore(player) + returnScore(player)
```

Comment:
One neat trick here is to use `switch`'s run-to-end behavior to our advantage

```JavaScript
switch (player.position) {
  case 'QB':
    return passingScore(player) + rushingScore(player)
  case 'RB':
  case 'WR':
    return rushingScore(player) + receivingScore(player) + returnScore(player)
  case 'TE':
    return receivingScore(player)
  default:
    return 0
}
```

If it matches on RB, since there is no break or return, it will keep running to WR which is good since they calculate the same way.